core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphqlCallAction.kt

```diff
@@ -0,0 +1,36 @@
+package org.evomaster.core.problem.graphql
+
+import org.evomaster.core.problem.graphql.param.GraphqlParam
+import org.evomaster.core.problem.rest.GraphqlAction
+import org.evomaster.core.search.Action
+import org.evomaster.core.search.gene.Gene
+
+class GraphqlCallAction (
+        /**
+         * Identifier unique within the individual
+         * **/
+        val id: String,
+        val functionName: String,
+        val parameters:  MutableList<GraphqlParam>,
```

Comment:
brainstorming here: a GraphQL query can be represented as JSON object. so we could use a single ObjectGene to represent it. or are there cases in which we might need several different type of parameters? I can only think of HTTP Header parameters

Reply:
I believe we just need one type of parameter for HTTP Header and one for the JSON object.